src/Utils/request/handleResponse.ts

```diff
@@ -14,6 +14,29 @@ export default function handleResponse(
 
   // 400/406 Bad Request
   if (res.status === 400 || res.status === 406) {
+    // Asset IP Conflict
+    if (
+      Array.isArray(error?.non_field_errors) &&
+      error.non_field_errors.length === 1 &&
+      error.non_field_errors[0].includes("IP Address")
+    ) {
+      const errorMsg = error.non_field_errors[0];
+
+      const ipAddress = errorMsg.split(" ")[2];
+
+      const facility = errorMsg
+        .substring(errorMsg.indexOf("by") + 3)
+        .split(" ")
+        .slice(0, -1)
+        .join(" ");
+
+      notify?.BadRequest({
+        errs: `IP Conflict: ${ipAddress} is in-use by '${facility}' asset within the facility`,
+      });
+
+      return;
+    }
+
```

Comment:
this logic is specific to asset configuration and not across all requests. Move this to be done in the `onResponse` of the request options and dispatch notification there instead of in the shared `handleResponse`

Reply:
hey @rithviknishad 

i did try but it says 
TS2339: Property status does not exist on type RequestResult<AssetData>

![image](https://github.com/coronasafe/care_fe/assets/98876115/70b4c07f-5740-4d31-a4ff-de13f2c0566c)

Reply:
res should be deconstructed from the params

```ts
onResponse: ({ res }) => { ... }
```